src/terra-dev-site/test/notification-banner/ExampleForAccessibilityTesting.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import NotificationBanner from '../../../notification-banner';
+
+const Example = ({ id }) => {
+  const [showErrorBanner, setShowErrorBanner] = useState(false);
+  const [showHazardMediumBanner, setShowHazardMediumBanner] = useState(false);
+  const [showUnsatisfiedBanner, setShowUnsatisfiedBanner] = useState(true);
+  const [showUnverifiedBanner, setShowUnverifiedBanner] = useState(false);
+  const [showHazardLowBanner, setShowHazardLowBanner] = useState(false);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setShowErrorBanner(true);
+    }, 10000);
+  }, []);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setShowHazardMediumBanner(true);
+    }, 30000);
+  }, []);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setShowUnverifiedBanner(true);
+    }, 60000);
+  }, []);
+
+  return (
+    <>
+      {showErrorBanner && <NotificationBanner variant="error" id={`error-banner-${id}`} description="Something happened..." />}
+      {showUnsatisfiedBanner && <NotificationBanner variant="unsatisfied" id={`unsatisfied-banner-${id}`} />}
+      {showHazardMediumBanner && (
+        <NotificationBanner
+          variant="hazard-medium"
+          id={`hazard-medium-banner-${id}`}
+          onRequestClose={() => setShowHazardMediumBanner(false)}
+        />
+      )}
+      {showUnverifiedBanner && (
+        <NotificationBanner
+          variant="unverified"
+          id={`unverified-banner-${id}`}
+          description={<div>There are records that have been included that need to be verified before they are officially added. Please review and ensure they should be included.</div>}
+          bannerAction={{
+            text: 'Verify Records',
+            onClick: () => {
+              alert('records verified.'); // eslint-disable-line no-alert
+              setShowUnverifiedBanner(false);
+            },
+          }}
+        />
+      )}
+      {showHazardLowBanner && <NotificationBanner variant="hazard-low" id={`hazard-low-banner-${id}`} />}
+      <p>The error banner will appear after 10 seconds.</p>
+      <button
+        onClick={() => setShowErrorBanner(!showErrorBanner)}
+        type="button"
+        id={`toggle-error-banner-${id}`}
+      >
+        Show/Hide Error Banner
```

Comment:
Could we flex the text of these buttons based on whether or not things are already hidden shown?  Could make the examples a little easier to interact with.

Reply:
At this time, I'd prefer to make this change in a follow up PR.